src/parser/shaman/elemental/modules/features/StormFireElemental.js

```diff
@@ -0,0 +1,55 @@
+import Analyzer from 'parser/core/Analyzer';
+import SPELLS from "common/SPELLS/shaman";
+import TALENTS from "common/SPELLS/talents/shaman";
+import SpellUsable from 'parser/shared/modules/SpellUsable';
+
+class StormFireElemental extends Analyzer {
+
+  static dependencies = {
+    spellUsable:SpellUsable,
+  };
+  elementalData = {
+    FireElemental: {
+      summon: SPELLS.FIRE_ELEMENTAL.id,
+      damageSpells: [
+        SPELLS.FIRE_ELEMENTAL_FIRE_BLAST.id,
+        SPELLS.FIRE_ELEMENTAL_METEOR.id,
+        SPELLS.FIRE_ELEMENTAL_IMMOLATE.id,
+      ],
+    },
+    StormElemental: {
+      summon: TALENTS.STORM_ELEMENTAL_TALENT,
+      damageSpells: [
+        SPELLS.EYE_OF_THE_STORM.id,
+        SPELLS.WIND_GUST.id,
+        SPELLS.CALL_LIGHTNING.id,
+      ],
+    },
+  };
+
+  last_pet_summon_timeStamp=null;
+
+  relevantData = this.selectedCombatant.hasTalent(TALENTS.STORM_ELEMENTAL_TALENT.id) ? this.data.StormElemental : this.data.FireElemental;
+
+
+  on_byPlayerPet_damage(event){
+      if(this.last_pet_summon_timeStamp!==null) {
+        return;
+      }
+      if(!this.relevant.damageSpells.includes(event.ability.guid)) {
+        return;
```

Comment:
Shouldn't this be called `!this.relevantData.damageSpells.includes(...)`

Reply:
done.
IntelliJ must have missed that for some reason